prime-router/metadata/schemas/covid-19.schema

```diff
@@ -27,7 +27,7 @@ elements:
     hl7Field: NTE-3
 
   - name: comment_source
-    type: CODE
+    type: ID
```

Comment:
I adjusted the fake data generator to use a value for the value set if a valueset is defined. Otherwise, it uses that number generator. There are a bunch of fields in the COVID-19 schema maked as TYPE: ID that I think should not be. So I'm going to make a new ticket to evaluate the TYPE usage in that schema to see what's really needed